### PR TITLE
Add solarwinds to root group, enable support for default statsd task (AO-12153)

### PR DESCRIPTION
**Pull request by gortyy**
With this changeset users will be able to load custom taskfiles with docker:
```shell
docker run -d -e APPOPTICS_TOKEN="<token>" \
           -v my_statsd.yaml:/opt/SolarWinds/Snap/etc/plugins.d/statsd.yaml \
           solarwinds/solarwinds-snap-agent-docker:1.0.1
```

https://swicloud.atlassian.net/browse/AO-12153

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,10 +1,10 @@
-TAG="1.0.0"
+TAG="1.0.1"
```
Comment (lootek) [inline Makefile]: maybe 1.1.0 as you've added new features?

Comment (gortyy) [reply Makefile]: that's right, fixed

**Comment by tomekkolo** [comment]
Please add comment in README on how to load custom task file

**Comment by gortyy** [comment]
@tomekkolo Usage info added to readme

**Pull request merged**